website/client/components/avatar.vue

```diff
@@ -16,22 +23,24 @@
     // Show avatar only if not currently affected by visual buff
     template(v-if!="!member.stats.buffs.snowball && !member.stats.buffs.spookySparkles && !member.stats.buffs.shinySeed && !member.stats.buffs.seafoam")
       span(:class="'chair_' + member.preferences.chair")
-      span(:class="getGearClass('back')")
+      span(v-tooltip="showEquipments", :class="getGearClass('back')")
       span(:class="skinClass")
       span(:class="member.preferences.size + '_shirt_' + member.preferences.shirt")
       span.head_0
       span(:class="member.preferences.size + '_' + getGearClass('armor')")
       span.head_0
-      span(:class="getGearClass('back_collar')")
-      span(:class="getGearClass('body')")
+      span(v-tooltip="showEquipments",:class="getGearClass('back_collar')")
+      span(v-tooltip="showEquipments",:class="getGearClass('body')")
       template(v-for="type in ['base', 'bangs', 'mustache', 'beard']")
         span(:class="'hair_' + type + '_' + member.preferences.hair[type] + '_' + member.preferences.hair.color")
-      span(:class="getGearClass('eyewear')")
-      span(:class="getGearClass('head')")
-      span(:class="getGearClass('headAccessory')")
+      span(v-tooltip="showEquipments",:class="getGearClass('eyewear')")
```

Comment:
if you pull the latest changes from develop, this should work without having to add new scripts